load_questions.dart

```diff
@@ -13,9 +13,12 @@ void main() async {
   File json = File('./quiz_app_questions.json');
   final questions = jsonDecode(json.readAsStringSync());
 
-  Database db = Database(client);
-  final collectionId = 'quiz_questions';
-  final res = await db.createCollection(
+  Databases db = Databases(client, databaseId: 'unique()');
```

Comment:
Maybe this one should be **default** also to match up with the other value after created

Reply:
I added this commit today morning after I read the discussion @stnguyen90 had with @2002Bishwajeet concerning how to create the databases with Serverside SDK. You have to provide a "unique()" parameter first and then provide the actual id to be used. If I'm not wrong since the constructor has to be there 😅

Reply:
Should I change it to `default` instead of `Quizy` ?

Reply:
Putting `unique()` and then calling `db.create(name: 'Quizy');` would create a Database with a unique auto generated ID, but you can also use your own custom ID to create the Database with that ID.

I agree with @brandonroberts suggestion.

Reply:
Okay. I'll fix that. Thanks for the clarification. Seems I got confused as you explained.